src/main/scala/org/apache/mesos/chronos/scheduler/jobs/Jobs.scala

```diff
@@ -101,7 +103,8 @@ case class ScheduleBasedJob(
                              @JsonProperty override val environmentVariables: Seq[EnvironmentVariable] = List(),
                              @JsonProperty override val shell: Boolean = true,
                              @JsonProperty override val arguments: Seq[String] = List(),
-                             @JsonProperty override val softError: Boolean = false)
+                             @JsonProperty override val softError: Boolean = false,
+                             @JsonProperty override val dataJob: Boolean = false)
   extends BaseJob
 
```

Comment:
I could add it as a type, but this field would now be an enum or a string.  Otherwise, I would not know what true or false meant.  @elingg thoughts?